index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+    <header>
+        <div class="nav-bar">
+            <div class="nav-logo">
+                <h1>My Logo</h1>
+            </div>
+            <nav class="nav-links">
+                <div class="ambuger">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </div>
+            </nav>
+        </div>
+    </header>
+    <main class="main-card">
+        <section>
+            <div class="main-container">
+                <div class="intro-text">
+                    <h2>I'm Emmanuel Glad to see you!</h2>
```

Comment:
- [ ] I think it would be better to use a ```<h1>``` on your project title to replace ```<h2>``` 😎😎 And  If you want that google (and other search engines) to know about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.